afl-fuzz.c

```diff
@@ -3440,7 +3449,8 @@ static void write_stats_file(double bitmap_cvg, double stability, double eps) {
              "afl_banner        : %s\n"
              "afl_version       : " VERSION "\n"
              "target_mode       : %s%s%s%s%s%s%s\n"
-             "command_line      : %s\n",
+             "command_line      : %s\n"
```

Comment:
Why was command_line added?

Reply:
`command_line` was already in the stats. I removed the trailing comma to append `slowest_exec_ms` at the end.

Reply:
O sorry for not paying attention. Yeah I even vaguely remembered command_line being in stats so I was a bit confused. 
Thanks for clarifying!